internal/users/users.go

```diff
@@ -0,0 +1,176 @@
+// Package users support all common action on the system for user handling.
+package users
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"log/slog"
+	"os"
+	"os/exec"
+	"slices"
+	"strings"
+
+	"github.com/ubuntu/decorate"
+)
+
+// UserInfo is the user information returned by the broker. We use that to build our own buckets content.
```

Comment:
> We use that to build our own buckets content.

 I feel like we need to remove the buckets reference, as it does not make sense out of the cache context.